### ISSUE TITLE
Use built-in <code> classNames to enhance the *Dependency graph* code snippets

### ISSUE THREAD

**Issue by AMBudnik** [handsontable/hyperformula#1361]
### Description
I think that we could use built-in `<code>` `className` to enhance the *Dependency graph* code snippets. Now, it is hard to read it if we switch from examples that use the coloring functionality to a one without it.

We can use that even if the language is not defined, so that is not a limitation.

### Screenshots
Colored syntax
![Zrzut ekranu 2023-12-28 o 09 06 06](https://github.com/handsontable/hyperformula/assets/10757813/71b35725-67a9-4140-a0b6-f1daa4265811)

Not colored
![Zrzut ekranu 2023-12-28 o 09 06 11](https://github.com/handsontable/hyperformula/assets/10757813/cef75786-2c3a-4375-b972-9100e53a9a16)

### Links
* *Dependency grapth* guidem - https://hyperformula.handsontable.com/guide/dependency-graph.html#cells-in-the-dependency-graph

### My environment
Browser: Chrome 120
System: MacOS Ventura
Browser Zoom: 100%
System Scale: 100%

**Comment by sequba** [comment]
This snippet is in pseudocode. How can we make it colored when there is no defined syntax for it?

**Comment by AMBudnik** [comment]
@sequba I get your point. But I still think that this would look a way better.

![Zrzut ekranu 2023-12-28 o 14 32 26](https://github.com/handsontable/hyperformula/assets/10757813/5adf3254-fb3a-4d7a-bd3e-d2edd8017664)


```
<code>
  <span class="token">AllCellDependents</span>=<span class="token variable">{'start'}</span>
  <span class="token keyword">let</span> <span class="token variable">Q</span> <span class="token keyword">be</span> <span class="token keyword">an</span> <span class="token keyword">empty</span> <span class="token keyword">queue</span>
  <span class="token variable">Q</span>.<span class="token function">enqueue</span>(<span class="token variable">start</span>)
  <span class="token keyword">while</span> <span class="token variable">Q</span> <span class="token keyword">is</span> <span class="token keyword">not</span> <span class="token keyword">empty</span> <span class="token keyword">do</span>
      <span class="token variable">cell</span> := <span class="token variable">Q</span>.<span class="token function">dequeue</span>()
      <span class="token variable">S</span> := <span class="token function">getCellDependents</span>(<span class="token variable">cell</span>)
      <span class="token keyword">for</span> <span class="token keyword">all</span> <span class="token variable">cells</span> <span class="token variable">c</span> <span class="token keyword">in</span> <span class="token variable">S</span> <span class="token keyword">do</span>:
          <span class="token keyword">if</span> <span class="token variable">c</span> <span class="token keyword">is</span> <span class="token keyword">not</span> <span class="token keyword">in</span> <span class="token variable">AllCellDependents</span> <span class="token keyword">then</span>:
              <span class="token function">insert</span> <span class="token variable">w</span> <span class="token keyword">to</span> <span class="token variable">AllCellDependents</span>
              <span class="token variable">Q</span>.<span class="token function">enqueue</span>(<span class="token variable">c</span>)

</code>
```

**Comment by sequba** [comment]
Ok, you meant coloring it manually. Yes, we can do that 👍

**Comment by sequba** [comment]
Released with version 2.6.2.
